components/brave_wallet_ui/stories/locale.ts

```diff
@@ -668,5 +668,16 @@ provideStrings({
   // Wallet Welcome Perks
   braveWalletPerksTokens: 'Buy, send, and swap 200+ crypto assets',
   braveWalletMultiChain: 'Multi-chain & NFT support',
-  braveWalletPerksBrowserNative: 'Browser-native, no risky extensions'
+  braveWalletPerksBrowserNative: 'Browser-native, no risky extensions',
+
+  // Portfolio Asset More Popup
+  braveWalletPortfolioTokenDetailsMenuLabel: 'Token details',
+  braveWalletPortfolioViewOnExplorerMenuLabel: 'View on block explorer',
+  braveWalletPortfolioHideTokenMenuLabel: 'Hide token',
+  braveWalletHideTokenModalTitle: 'Hide token',
+
+  // Token detail modals
+  braveWalletMakeTokenVisibleInstructions: 'You can make this asset visible in the future by going to "Visible assets" portfolio tab',
```

Comment:
Shouldn't this be something like 'You can make this asset visible again in the future by clicking the "+ Visible assets" button at the bottom of the "Portfolio" tab'

Reply:
Fixed here: e764aecb16be39852d678141a5b27c75933e04cd